_pages/en_US/bannerbomb3-fredtool-(twn).txt

```diff
@@ -125,21 +125,23 @@ If you would prefer a visual guide to this section, one is available [here](http
 1. Press (A), then press START and SELECT at the same time to begin the process
 1. Once completed and the bottom screen says “done.”, exit b9sTool, then power off your device
     + You may have to force power off by holding the power button
-    + If you see the Luma Configuration screen, continue with the guide without powering off
+    + If your device shuts down when you try to power it on, ensure that you have copied `boot.firm` from the Luma3DS `.zip` to the root of your SD card
+    + If you see the Luma Configuration screen, press (Start) to reboot, then continue with the guide
 
-#### Section VI - Configuring Luma3DS
+#### Section VI - Luma3DS Verification
```

Comment:
is there any situation in which the uesr wouldn't see the luma configuration screen after installing luma, that would neccesitate this section?

Reply:
B9STool used to (in rare cases) silently fail to install B9S, hence the [special troubleshooting for it](https://github.com/zoogie/b9sTool/blob/master/TROUBLESHOOTING.md) (see section B). Not sure if this still happens, and also not sure if B9STool consistent And obviously if B9S wasn't installed then the user wouldn't see the Luma configuration menu/etc.